onnxruntime/python/tools/transformers/models/phi2/README.md

```diff
@@ -0,0 +1,115 @@
+# Phi2 Optimizations
+## Prerequisites
+```bash
+pip install -r requirements.txt
+```
+_From source:_
+```bash
+pip install onnxruntime-gpu==1.17.0 # or onnxruntime==1.17.0 if using cpu
+git clone git@github.com:microsoft/onnxruntime.git
+cd onnxruntime/onnxruntime/python/tools/transformers
+python -m models.phi2.convert_to_onnx -h
+```
+_From wheel:_ \
+Install [ORT nightly package]([https://onnxruntime.ai/docs/install/](https://onnxruntime.ai/docs/install/#inference-install-table-for-all-languages))
+```bash
+python -m onnxruntime.transformers.models.phi2.convert_to_onnx -h
+```
+
+## Export optimized phi2 onnx model for different scenarios
+**Export FP32 ONNX model for Nvidia GPUs** \
+_From source:_
+```
+python -m models.phi2.convert_to_onnx --fp32_gpu
+```
+_From wheel:_
+```
+python -m onnxruntime.transformers.models.phi2.convert_to_onnx --fp32_gpu
+```
+\
+**Export FP16 ONNX model for Nvidia GPUs** \
+_From source:_
+```
+python -m models.phi2.convert_to_onnx --fp32_gpu
+```
+_From wheel:_
+```
+python -m onnxruntime.transformers.models.phi2.convert_to_onnx --fp32_gpu
+```
+\
+**Export INT4 ONNX model for Nvidia GPUs** \
+_From source:_
+```
+python -m models.phi2.convert_to_onnx --fp32_gpu
+```
+_From wheel:_
+```
+python -m onnxruntime.transformers.models.phi2.convert_to_onnx --fp32_gpu
+```
+\
+**Export FP16 ONNX model for Nvidia A100** \

```

Comment:
for Nvidia GPUs with [compute compacity](https://developer.nvidia.com/cuda-gpus) 8.*